lib/navigation_bar_controller.dart

```diff
@@ -48,8 +48,9 @@ class _AppBottomNavBarControllerState extends State<AppBottomNavBarController> {
         openPage: openPage,
       ),
       SearchPage(key: PageStorageKey('searchservices')),
-      ProfilePage(key: PageStorageKey('yourprofile')),
```

Comment:
`ProfilePage` is not a page anymore. It now is a separate screen that will be opened by `Navigator.push()`